docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -1,8 +1,8 @@
 # Get invited and onboard to TechPass
 
-This article guides you how to request for a TechPass account and onboard to it. Refer to the flow chart on [Onboard to TechPass](onboard-to-techpass) page to know if you can sign up for TechPass account via TechPass portal or not. If you can't sign up for TechPass account via TechPass portal, you need to request to get invited.
+This article walks you through the TechPass portal sign-up and onboarding process. Check the flow chart on the [Onboard to TechPass](onboard-to-techpass) page to see if you're eligible to sign up via the portal. If you can't sign up for TechPass account via TechPass portal, you need to request to get invited.
 
-?> **Note**<br>- Alternatively, public officers with a non-SE GSIB device can access [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg) to request for TechPass and SEED(optional) provisioning. <br>- For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+?> **Note**<br>- Alternatively, public officers with a non-SE GSIB device can access [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg) to request for TechPass and SEED(optional) provisioning. <br>- For more details, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
```

Comment:
A bit of a verb agreement here. Can i suggest
"Alternatively, a public officer with a.. "
https://learn.microsoft.com/en-us/style-guide/grammar/verbs#verb-agreement

can i suggest ... for "TechPass and SEED provisioning, which is optional." 

since we are here is it SEED(optional) or SEED (optional)?